### PR TITLE
feat: add force_new_deployment variable

### PR DESCRIPTION
**Pull request by syphernl**
## what
* Added the `force_new_deployment` flag to force new task deployment of the service
* Updated github configs

## why
* To make it possible to force a new deployment

## references
* https://github.com/cloudposse/terraform-aws-ecs-alb-service-task/pull/112

### CODE REVIEW

File: .github/workflows/validate-codeowners.yml [hunk 1]
```diff
@@ -1,7 +1,5 @@
 name: Validate Codeowners
 on:
-  workflow_dispatch:
-
```
Comment (Nuru) [inline .github/workflows/validate-codeowners.yml:4]: You probably did not mean to delete this, but in any case, the `workflow_dispatch` trigger needs to remain

**Review by Nuru — CHANGES_REQUESTED** [review]

**Review by nitrocode — DISMISSED** [review]

**Review by nitrocode — APPROVED** [review]

**Review by Gowiem — APPROVED** [review]

**Pull request merged**